frontend/src/main.jsx

```diff
@@ -35,9 +41,8 @@ const router = createBrowserRouter([
     ],
   },
 ]);
-
 ReactDOM.createRoot(document.getElementById("root")).render(
-  <React.StrictMode>
+  <AuthProvider>
```

Comment:
You already have AuthProvider in the router, so please remove this one. Also bring back the React.StrictMode as it can help catch bugs during development